### ISSUE TITLE
Plugin hangs at processing

### ISSUE THREAD

**Issue by frankmeeuwsen** [nhaouari/obsidian-textgenerator-plugin#9]
I installed the latest version of the plugin, I've updated Obsidian to the latest version. 
I tried one of your examples with the frontmatter. When I want to generate text, the statusbar hangs on Text Generator(160):processing...
I restarted the app a few times but no luck. I also made a brand new vault without other plugins but still the same behaviour. Is there a way to debug this? I am a bit familiair with the developers tools and network window, but I'm not quite sure where to pay attention to.

Thanks

**Comment by nhaouari** [comment]
Thanks for letting me know. Could you please try opening the Developer Tools console (Ctrl+Shift+i) and seeing if there are any errors being displayed there? That might give us a clue as to what's going on.

**Comment by frankmeeuwsen** [comment]
The moment I do "Generate text" I get this error in the console:

```
Uncaught (in promise) Error: Request failed, status 429
    at new t (app.js:1:722668)
    at Rw (app.js:1:722860)
    at app.js:1:723537
    at app.js:1:235237
    at Object.next (app.js:1:235342)
    at a (app.js:1:234081)
t @ app.js:1
Rw @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
a @ app.js:1
Promise.then (async)
step @ plugin:obsidian-textgenerator-plugin:62
eval @ plugin:obsidian-textgenerator-plugin:63
__async @ plugin:obsidian-textgenerator-plugin:47
editorCallback @ plugin:obsidian-textgenerator-plugin:210
e.mobileOnly.Fw.isMobile.e.checkCallback @ app.js:1
CB @ app.js:1
t.onChooseItem @ app.js:1
t.onChooseSuggestion @ app.js:1
t.selectSuggestion @ app.js:1
e.useSelectedItem @ app.js:1
(anonymous) @ app.js:1
e.handleKey @ app.js:1
e.onKeyEvent @ app.js:1```

**Comment by frankmeeuwsen** [comment]
Hmm... I think I found it already. I made an account at OpenAI. I did this with my Google mail. I had no idea I ever made an account at OpenAI before (I don't know about you, but I try dozens of tools...). So I already spent my free credits there. When I made a new account with a new mail addres, I had to verify my phonenumber. That's when I saw the message I already ran out of free credits because of an earlier account. I think that's the problem. Maybe have some callback in the screen or statusbar to check on the credits?
